astroquery/gemini/core.py

```diff
@@ -106,6 +111,28 @@ def __init__(self, *args):
         """
         super().__init__()
 
+    def _login(self, username, password):
```

Comment:
ideally this would also need to be tested, but I suppose we need to have a dummy account for testing purposes to do that which points beyond this PR, so I don't hold up merging on its account.

Reply:
I'll see if they are willing to add an unprivileged test user for astroquery testing.  Is it possible to store the password outside of the git tree?

Reply:
secrets can be stored in the github repo admin interface, I'll investigate whether those could be used for this purpose.